src/client/components/Molecules/HeroSearchForm/HeroSearchForm.jsx

```diff
@@ -0,0 +1,26 @@
+import { Link } from 'react-router-dom';
+
+import css from './HeroSearchForm.module.css';
+
+
+const HeroSearchForm = ({ value, onChange }) => {
+    return (
+      <form className={css.formSearch}>
+        <div className={css.formSearchLabel}>
+          <input
+            type="text"
+            value={value}
+            onChange={e => onChange(e.target.value)}
+            placeholder="Beef"
+            className={css.formSearchInput}
+          ></input>
+          <Link className={css.formSearchButton} to="/search">
+            Search
+          </Link>
+        </div>
+      </form>
+    );
+  };
+  
+  export default HeroSearchForm;
```

Comment:
we are using name export convention like so

f.ex. 
`export { variable }`

Keep it in mind for rest code.

Reply:
❓